secscanner2junit/sast.py

```diff
@@ -63,8 +65,14 @@ def parse_findings(self, finding, time):
             pass
 
         f_type = finding['identifiers'][0]['name']
+        
+        try:
+            finding_id = properties['id']
+        except KeyError:
+            finding_id = str(random.randrange(1, 10000000))
+        
         if properties['name']:
-            tc = TestCase(name=properties['name'] + " " + properties['id'] , classname=self.p_type, file=properties['file'], elapsed_sec=time, line=properties['start_line'])
+            tc = TestCase(name=properties['name'] + " " + finding_id , classname=self.p_type, file=properties['file'], elapsed_sec=time, line=properties['start_line'])
```

Comment:
When we attach the internal id like this, it will be shown in this way:

<img width="853" alt="Screenshot 2022-08-05 at 09 05 13" src="https://user-images.githubusercontent.com/16398152/183021705-17d87f52-de26-4eb7-a940-4f4c366f5ba6.png">

What do you think about rather displaying it like this?
<img width="1088" alt="Screenshot 2022-08-05 at 09 15 12" src="https://user-images.githubusercontent.com/16398152/183023177-e26739ee-f681-428d-a6b7-14a1933ca700.png">

Reply:
Done :)